plugin/lf.vim

```diff
@@ -30,10 +30,10 @@ else
 endif
 
 function! OpenLfIn(path, edit_cmd)
-  let currentPath = shellescape(expand(a:path))
+  let currentPath = shellescape(fnamemodify(expand(a:path), ":p:h"))
```

Comment:
Not at computer, but doesn’t this remove any file basename? Is the current file still highlighted when opening Lf?

Reply:
Well, `OpenLfIn()` is used when opening a directory or using any of the `Lf` commands (which opens a directory), so it always gets a directory.
```
Thus ":p:h" on a directory name results on the directory name itself (without trailing slash).
```

When they want to open a file using lf, they would have to use the `Floaterm lf` command and that goes straight into to wrapper.

> Is the current file still highlighted when opening Lf?

So yes.

Reply:
Fixed.